src/main/java/de/tum/cit/aet/artemis/core/service/ArtemisSuccessfulLoginService.java

```diff
@@ -48,6 +48,10 @@ public class ArtemisSuccessfulLoginService {
 
     private final MailSendingService mailSendingService;
 
+    /**
+     * Ensures that the password reset links for both English and German are initialized properly.
+     * If the configured links are empty or set to a placeholder, it uses the default link the Artemis server URL/account/reset/request.
```

Comment:
I stumbled a bit across it and I know what is happening there, maybe the formulation could be improved 😅 
```suggestion
     * If the configured links are empty or set to a placeholder, it uses the default link, the ArtemisServerURL/account/reset/request.
```